tests/test_trnsys.py

```diff
@@ -23,6 +32,48 @@ def step_forward(self, steps: int) -> (bool, int):
             return (False, 1)
         return (True, 0)
 
+    def get_output_value(self, unit: int, output_number: int) -> (float, int):
```

Comment:
In seeing this, I don't think I'm in love with returning a tuple. I think a namedtuple is the way to go here (not a dataclass) mainly because it is immutable and unpackable.

Reply:
I like that idea a lot.  Implemented in 0bf6fba